src/c-writer.cc

```diff
@@ -2712,6 +2940,101 @@ void CWriter::Write(const Func& func) {
   func_ = nullptr;
 }
 
+void CWriter::WriteTailCallee(const Func& func) {
+  Stream* prev_stream = stream_;
+  BeginFunction(func);
+  in_tail_callee_ = true;
+  PushFuncSection();
+  WriteTailCallFuncDeclaration(GetTailCallRef(func.name));
+  Write(" ", OpenBrace());
+  WriteTailCallAsserts(func.decl.sig);
+  Write(ModuleInstanceTypeName(), "* instance = *instance_ptr;", Newline());
+
+  std::vector<std::string> index_to_name;
+  MakeTypeBindingReverseMapping(func_->GetNumParamsAndLocals(), func_->bindings,
```

Comment:
The mix of `func` and `func_` in the same function is a bit confusing. From what I can see, they are the same obj? If these are the same could be use either one. If different, could we rename one to be more descriptive?

Reply:
Fixed in 7a08fc17285ff5276896d85475b798aaae4da856 to use "func" consistently (matching the existing `CWriter::Write(const Func&)`).